### ISSUE TITLE
Issues with new theme

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#442]
@domdomegg @neslihanturan @maskaravivek I've been doing more in-depth testing before releasing the new version (on a Nexus S emulator API 23 and 25, prior to real device testing), and a few things that I noticed with the new themes/icons:

1. When sent to the overflow menu due to the action bar being full, "From gallery" shows up with its icon on the left. Besides being the only item in the overflow menu with an icon, when the light theme is selected, the icon isn't visible due to being white.

![gallery](https://cloud.githubusercontent.com/assets/3611199/24091641/027a846e-0d95-11e7-8151-80d50df47166.png)

2. The subtext in the tutorial is black now, while the text is white. IMO keeping the text and subtext roughly the same color is a lot more aesthetic.

![tutorial2](https://cloud.githubusercontent.com/assets/3611199/24091768/d9ce91da-0d95-11e7-8a44-5d0855d0d210.png)

3. I think it might be a good idea to make the dark theme the default instead of the light theme, so that people start out with the theme that they are used to, and have the option to change it if they prefer light. What do you guys think?

4. Very minor issue, and could just be a personal preference, but in the media details pane, landscape images have a large white band above them. IMO this looks okay in the dark theme, but rather jarring in the light theme. 

![media details](https://cloud.githubusercontent.com/assets/3611199/24091755/bfba3bbe-0d95-11e7-9f53-7d8548d9bdd3.png)

**Comment by domdomegg** [comment]
I agree with all the points except for 4, I think it's fine. However it may be an idea to change that layout of the media_detail_fragment to something more material-ly at some point where we could change the vertical alignment of the image.

Might be good to get some feedback from beta testers on the new theme as well, perhaps vote on some issues.

**Comment by misaochan** [comment]
Yeah, I don't think 4 is a big issue either. Would be good if 1-3 could be fixed before we start publicizing though.

Having a polling option sounds good too. Will make a separate issue for that.

**Comment by neslihanturan** [comment]
It seems like 2 is already solved by @veyndan . I can solve 1 and 3 but I am having problem about reproducing the case 1. Can you share a scenario @misaochan ?

**Comment by misaochan** [comment]
Thanks @veyndan !

@neslihanturan Case 1 happens when you have a small device (like the Nexus S that I emulated). When the action bar is full, extra items go to the overflow menu, which is when the issue occurs. Test it on a Nexus S emulator and see?

**Comment by misaochan** [comment]
Issue 5: When in light theme, uploading an image with a dark background makes it impossible to read black category text.

![screenshot_20170321-010026](https://cloud.githubusercontent.com/assets/3611199/24105832/dc4e7236-0dd1-11e7-835e-fc488de67042.png)

**Comment by neslihanturan** [comment]
I am on issue 5

**Comment by neslihanturan** [comment]
I recognize another one about spinner theme. Fixing now.

**Comment by veyndan** [comment]
With point no. 1, is it even necessary to have an icon in the overflow menu? As noted before, it feels a bit out of place considering it is the only item with an icon.

**Comment by misaochan** [comment]
Well-spotted @neslihanturan , thanks!

@veyndan I'm fine either way (with removing the icon when it's in overflow or changing its color).

**Comment by neslihanturan** [comment]
@veyndan I am okay with either way too. However, gallery menu has higher importance than other menus when it is in overflow (to upload). So it can be a good idea to show its icon to make user recognize it easier. If you decide to change it, I am online for a new PR.

**Comment by veyndan** [comment]
I'll make a new PR. I think a better way to show importance is hierarchically, by showing the gallery menu at the top of the overflow.
